content/en/network_monitoring/devices/profiles.md

```diff
@@ -64,6 +64,29 @@ Profiles can be used interchangeably, meaning devices that share MIB dependencie
 
 For more Datadog provided profiles, see the [GitHub repository][3].
 
+### Metadata definition by profile
+
+Profiles have a metadata section that can be used to define where and how metadata should be collected. Values can be static or come from an OID value.
+Supported fields can be found [here][4].
+
+With Datadog Agent version 7.52 and later, a new `device_type` field is supported as a device metadata. This can be set manually in the profile and can be used to filter on specific types of devices. Accepted values include:
```

Comment:
```suggestion
With Datadog Agent version 7.52 and later, there is a `device_type` field for device metadata. This can be set manually in the profile and can be used to filter on specific types of devices. Accepted values include:
```